view-creator-framework/build.gradle.kts

```diff
@@ -52,21 +52,21 @@ dependencies {
     implementation("org.apache.calcite:calcite-babel:1.34.0")
     implementation("com.google.code.gson:gson:2.10.1")
     implementation("org.apache.spark:spark-launcher_2.12:3.4.1")
-    implementation("org.xerial.snappy:snappy-java:1.1.10.1")
+    implementation("org.xerial.snappy:snappy-java:1.1.10.4")
     implementation("com.google.protobuf:protobuf-java-util:3.16.3")
     implementation("org.codehaus.janino:janino:3.1.9")
   }
   implementation(platform("io.netty:netty-bom:4.1.94.Final"))
   implementation(platform("org.glassfish.jersey:jersey-bom:2.40"))
   implementation(platform("org.jetbrains.kotlin:kotlin-bom:1.6.21"))
 
-  compileOnly("org.projectlombok:lombok:1.18.24")
-  annotationProcessor("org.projectlombok:lombok:1.18.24")
-  implementation("org.slf4j:slf4j-api:1.7.36")
-  implementation("com.typesafe:config:1.4.1")
+  compileOnly("org.projectlombok:lombok:1.18.26")
+  annotationProcessor("org.projectlombok:lombok:1.18.26")
+  implementation("org.slf4j:slf4j-api:2.0.5")
```

Comment:
After this change, hypertrace/hypertrace-ingestor transitively loads `slf4j-api:2.0.5` but is still on `log4j-slf4j-impl` instead of `slf4j2`. That led to failure of integration tests in QS.

Reply:
Raised PR to fix it: https://github.com/hypertrace/hypertrace-ingester/pull/425